docs/source/apps/mp-api.md

```diff
@@ -1,14 +1,19 @@
-# MarketPlace API
+# Standard API
 
-To declare which capabilities are supported by your application, a REST-API that complies with the MarketPlace specification must be set for your application.
-The API should be expressed using the [OpenAPI specification](https://www.openapis.org/) standard and be sent to the MarketPlace platform upon [registration](registration.md).
+The MarketPlace defines a standard API through which the registered application can be communicated and monitored via the MarketPlace.
+The API is defines a set of of endpoints or _capabilities_ to specify actions which MarketPlace applications may perform.
+An application provider may choose to support any subset of these capabilities, depending on the relevant functionalities.
 
-You can browse the official specification with redoc <a href="api.html" target="_blank">here</a>.
-The complete specification is hosted publicly on [GitHub](https://github.com/materials-marketplace/standard-app-api).
+The complete specification is hosted publicly on GitHub:<br>
```

Comment:
```suggestion
The complete specification is hosted publicly on GitHub:

```

Reply:
I think it actually looks better with the line break.

Reply:
Looks like there might be other options, but [this is the way to go ](https://www.markdownguide.org/basic-syntax/#line-break-best-practices)